inc/field/fieldsfield.class.php

```diff
@@ -152,16 +149,23 @@ public function getBlocks(): array {
    }
 
    public function showForm(array $options): void {
+      if (!\Plugin::isPluginActive('fields')) {
+         $template = '@formcreator/field/undefinedfield.html.twig';
+         TemplateRenderer::getInstance()->display($template, [
+            'item' => $this->question,
+            'params' => $options,
+         ]);
+         return;
+      }
```

Comment:
show an template for an undefined field as the plugin Additional Fields is not available and some HTML inputs cannot be rendered